### ISSUE TITLE
ImportError

### ISSUE THREAD

**Issue by Perfy** [madMAx43v3r/chia-gigahorse#42]
chia-gigahorse-farmer-1.6.2.giga7-windows.zip 
after run  start_farmer.cmd
![2023-03-07_175559](https://user-images.githubusercontent.com/198541/223631191-3862594d-1544-4e05-8d7e-5d45d1b0a2f8.png)

**Comment by EnamyX** [comment]
Did you manage to solve?
Same problem. This is not related to LTSC
I have 2 identical computers, everything works on one, exactly the same inscription on the second.

**Comment by EnamyX** [comment]
вобщем нашел решение. Такая проблема происходит на компьютере где нет подходящей видеокарты, или нет ее вообще. После установки видеокарты 1050, все запустилось без проблем.

**Comment by madMAx43v3r** [comment]
Try installing latest MSVC++ redistributable: https://aka.ms/vs/17/release/vc_redist.x64.exe

**Comment by Chiliheld** [comment]
Same Problem her on Win10 with installed vc-redist
@EnamyX have you solved this?

**Comment by EnamyX** [comment]
Да, проблема в ВИДЕОКАРТЕ. Не работает на видеокартах АМД(на любых) и если у карты меньше 6гб памяти(для с8 плотов).

**Comment by chuwis** [comment]
Daemon not started yet
Starting daemon
chia_harvester: Already running, use `-r` to restart
chia_farmer: Already running, use `-r` to restart
chia_full_node: Already running, use `-r` to restart
chia_wallet: started
dont start

**Comment by madMAx43v3r** [comment]
how you know it didn't start?

**Comment by chuwis** [comment]
> how you know it didn't start?
Failed to open parcels 17 plots
